src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -3,184 +3,245 @@
-              defaultMessage: 'Create a workspace',
-            })}
-          </h2>
-        </EuiTitle>
-        <EuiText size="s">
-          {i18n.translate('workspace.initial.createWorkspace.describe', {
-            defaultMessage: 'Organize projects by use case in a collaborative workspace',
-          })}
-        </EuiText>
+        <EuiFlexGroup direction="row" justifyContent="spaceBetween" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+              <EuiIcon type="wsSelector" size="m" />
+              &nbsp;
+              <EuiTitle size="m">
+                <h2>
+                  {i18n.translate('workspace.initial.title', {
+                    defaultMessage: 'My workspaces',
+                  })}
+                </h2>
+              </EuiTitle>
+            </div>
+            <EuiText size="s">
+              {i18n.translate('workspace.initial.createWorkspace.describe', {
+                defaultMessage:
+                  'Collaborate on use-case based projects with workspaces. {hasWorkspace, select, true { Select a workspace to get started.} false {}}',
+                values: { hasWorkspace: workspaceList.length > 0 },
+              })}
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>{isDashboardAdmin && createWorkspacePopover}</EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiFlexGroup justifyContent="spaceBetween" gutterSize="m">
+          {useCaseCards}
+        </EuiFlexGroup>
       </EuiFlexItem>
-      <EuiFlexItem grow={false}>{cards}</EuiFlexItem>
       <EuiFlexItem grow={false}>
-        <EuiFlexGroup direction="row" justifyContent="spaceBetween" alignItems="center">
-          <EuiFlexItem grow={false}>{isDashboardAdmin ? createButton : noAdminText}</EuiFlexItem>
-          <EuiFlexItem grow={false} style={{ maxWidth: '540px' }}>
-            <EuiPanel color="subdued" paddingSize="s" hasShadow={false} hasBorder={false}>
-              <EuiText size="s">
-                <EuiIcon type="dashboardApp" size="l" />
-                &nbsp;&nbsp;Explore live demo environment at{' '}
-                <EuiLink
-                  href="https://playground.opensearch.org/"
-                  external
-                  style={{ fontWeight: 'normal' }}
-                >
-                  playground.opensearch.org
-                </EuiLink>
-              </EuiText>
-            </EuiPanel>
+        <EuiFlexGroup justifyContent="spaceBetween" gutterSize="m">
+          <EuiFlexItem grow={false}>
+            <EuiText size="s" style={{ display: 'flex', alignItems: 'center' }}>
+              <EuiIcon type="reporter" size="s" color="primary" />
+              &nbsp;
+              <EuiLink
+                href="https://docs.aws.amazon.com/opensearch-service/latest/developerguide/what-is.html"
```

Comment:
Should it be the doc link related to workspace?

Reply:
I will confirm the link with pm again.

Reply:
The link is related to https://opensearch.org/docs/latest/about/

Reply:
The link is related to https://opensearch.org/docs/latest/about/